Assets/Scenes/PlayerVR.unity

```diff
@@ -0,0 +1,777 @@
+%YAML 1.1
```

Comment:
I'm not a fan of introducing a second Unity scene. See discussion [here](https://cvmlp.slack.com/archives/C05HWV5UZ1A/p1707154210737549?thread_ts=1707148506.826349&cid=C05HWV5UZ1A).

Reply:
As discussed, I believe that storing all application-specific parameters in a scene, and selecting which one to build in the `Build Settings` is easier to handle, maintain and extend as opposed to having one scene with extra build-specific tweaks to do prior to building.

Happy to discuss more.

Reply:
Discussion continues in the [slack thread](https://cvmlp.slack.com/archives/C05HWV5UZ1A/p1707154210737549?thread_ts=1707148506.826349&cid=C05HWV5UZ1A)